src/index.js

```diff
@@ -115,6 +115,7 @@ export default class ModalFilterPicker extends Component {
       return (
         <ListView
           enableEmptySections={false}
+          keyboardShouldPersistTaps={this.props.keyboardShouldPersistTaps?this.props.keyboardShouldPersistTaps:'never'}
```

Comment:
Please extract `keyboardShouldPersistTaps` in the props above so that this code here can be simplified to:

```
keyboardShouldPersistTaps || 'never'
```